Rakefile

```
@@ -138,3 +138,22 @@ task check_format: [:statichtml] do
     raise res.sort.uniq.join("\n")
   end
 end
+
+desc 'Check unnecessary indentation in samplecode'
+task :check_indent_in_samplecode do
+  errors = []
+  `git grep -F --name-only '\#@samplecode'`.lines(chomp: true).each do |path|
+    lines = File.read(path).lines(chomp: true)
+    lines.each.with_index do |line, idx|
+      next unless line.start_with?('#@samplecode')
+      next unless lines[idx + 1].start_with?(' ')
+
+      errors << "#{path}:#{idx + 1}: \#@samplecode の中に不要なインデントがあります。削除してください"
```

` + 1` のこしておきたい気もするし `with_index(1)` でいい気もしています

`with_index(1)` するなら

```suggestion
    lines.each.with_index(1) do |line, idx|
      next unless line.start_with?('#@samplecode')
      next unless lines[idx].start_with?(' ')

      errors << "#{path}:#{idx}: \#@samplecode の中に不要なインデントがあります。削除してください"
```

しないなら `each_with_index` でいいのかなあ

```suggestion
    lines.each_with_index do |line, idx|
      next unless line.start_with?('#@samplecode')
      next unless lines[idx + 1].start_with?(' ')

      errors << "#{path}:#{idx + 1}: \#@samplecode の中に不要なインデントがあります。削除してください"
```

---

`with_index(1)`しました